data/soapvalidator/Prefs/OutOfOffice/OOOPreventive.xml

```diff
@@ -0,0 +1,192 @@
+<t:tests xmlns:t="urn:zimbraTestHarness" xmlns:admin="urn:zimbraAdmin" xmlns:mail="urn:zimbraMail" xmlns:acct="urn:zimbraAccount">
+
+    <!-- Define properties for accounts -->
+    <t:property name="account1.name" value="account1.${TIME}.${COUNTER}@${defaultdomain.name}"/>
+    <t:property name="account2.name" value="account2.${TIME}.${COUNTER}@${defaultdomain.name}"/>
+    <t:property name="account3.name" value="account3.${TIME}.${COUNTER}@externaldomain.com"/>
+    <t:property name="alias.name" value="Alias.${TIME}.${COUNTER}@${defaultdomain.name}"/>
+    <t:property name="defaultpassword.value" value="z1mbr4@123"/>
```

Comment:
No need to define this property as can be inherit from global.properties

Reply:
Removed